pandas/core/groupby.py

```diff
@@ -3463,35 +3406,42 @@ class DataFrameGroupBy(NDFrameGroupBy):
 
     _block_agg_axis = 1
 
-    def __getitem__(self, key):
-        if self._selection is not None:
-            raise Exception('Column(s) %s already selected' % self._selection)
+    @Substitution(name='groupby')
+    @Appender(SelectionMixin._agg_doc)
+    @Appender(SelectionMixin._see_also_template)
+    def aggregate(self, arg, *args, **kwargs):
```

Comment:
The order of the doc appends seems wrong

I get the 'See also' part in the beginning:

```
In [6]: pd.core.groupby.DataFrameGroupBy.aggregate?
Signature: pd.core.groupby.DataFrameGroupBy.aggregate(self, arg, *args, **kwargs)
Docstring:
See also
--------
:func:`pandas.Series.groupby`
:func:`pandas.DataFrame.groupby`
Aggregate using input function or dict of {column -> function}

Parameters
----------
arg : function or dict
    Function to use for aggregating groups. If a function, must either
    work when passed a DataFrame or when passed to DataFrame.apply. If
    passed a dict, the keys must be DataFrame column names.

    Accepted Combinations are:
```